app/models/concerns/geo_module.rb

```diff
@@ -0,0 +1,11 @@
+module GeoModule
+  extend ActiveSupport::Concern
+  included do
+    def generate_geohash
+      if(self.latitude && self.longitude)
```

Comment:
__[rubocop]__
* Space after keyword `if` is missing.
* Redundant `self` detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-self-unless-required)